packages/create-svelte/shared/+eslint+prettier+typescript/.eslintrc.cjs

```diff
@@ -1,20 +1,34 @@
 module.exports = {
 	root: true,
+	extends: [
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended',
+		'plugin:svelte/recommended',
+		'prettier'
+	],
 	parser: '@typescript-eslint/parser',
-	extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'prettier'],
-	plugins: ['svelte3', '@typescript-eslint'],
+	plugins: ['@typescript-eslint'],
 	ignorePatterns: ['*.cjs'],
-	overrides: [{ files: ['*.svelte'], processor: 'svelte3/svelte3' }],
-	settings: {
-		'svelte3/typescript': () => require('typescript')
-	},
 	parserOptions: {
 		sourceType: 'module',
-		ecmaVersion: 2020
+		ecmaVersion: 2020,
+		extraFileExtensions: ['.svelte']
 	},
 	env: {
 		browser: true,
 		es2017: true,
 		node: true
-	}
+	},
+	overrides: [
+		{
+			files: ['*.svelte'],
+			parser: 'svelte-eslint-parser',
+			parserOptions: {
+				parser: {
+					ts: '@typescript-eslint/parser',
+					js: 'espree'
+				}
```

Comment:
The current configuration uses '@typescript-eslint/parser' as the parser for `*.js` files, so I think it makes sense to always use `@typescript-eslint/parser` as parser for `<script>` in `*.svalte` files.

```suggestion
				parser: '@typescript-eslint/parser'
```

Reply:
Ah yeah, I updated. Thank you!

Reply:
If that's the case then the `package.json` needs to be adjusted to always instill this parser (and then we can probably also remove `shared/+eslint+typescript`).
Just curious: why is that parser used either way?

Reply:
Maybe I didn't understand your comment properly but I set `@typescript-eslint/parser` for only TypeScript projects right?
There are 4 `.eslintrc.cjs` files and 2 files are for TypeScript project.
So I set `@typescript-eslint/parser` for only these 2 projects.

And I think already `package.json` for TypeScript & ESLint project has `@typescript-eslint/parser`.

Reply:
Oh, sorry for the noise then, I misread the comment and didn't look at the code close enough - all good 👍